template.yaml

```diff
@@ -335,6 +335,41 @@ Resources:
 
   #endregion
 
+  #region Fetch JWKS Lambda
+
+  FetchJwksFunction:
+    Type: AWS::Serverless::Function
+    # checkov:skip=CKV_AWS_115: Reserved concurrency will be looked into in ATO-657
+    # checkov:skip=CKV_AWS_116: DLQ is not required
+    # checkov:skip=CKV_AWS_117: Lambda needs to exist outside our VPC so default AWS Lambda service VPC is OK
```

Comment:
I've assumed it's OK not to explicitly define a VPC for the lambda and just let exist in the default location which has public internet access
https://docs.aws.amazon.com/lambda/latest/operatorguide/networking-vpc